mocks/tests/test_mocks.py

```diff
@@ -82,9 +83,9 @@ def test_scale_light_by_distance(particle_positions_3D,
     assert type(luminosity) == u.Quantity
     assert positions.shape == (5, 3)
     assert luminosity.shape == (5,)
-    assert positions.max() == pytest.approx(19.060319816048057, rel=1e-12)
-    assert luminosity.value.max() == pytest.approx(2.047694415864918,
-                                                   1e-12)
+    assert positions.max() == pytest.approx(18.999980867240506, rel=1e-12)
+    assert luminosity.value.max() == pytest.approx(2.0407101,
+                                                   1e-6)
```

Comment:
A bit curious as to why the results of all of these test calculations have changed slightly. All of the input parameters should be the same as before, shouldn't they?

Reply:
I have customized the cosmology and to check if it has been successful, I had to give some values other than the default cosmology. This differences in the cosmological parameters affected the calculated values.

Reply:
And I should mention that there are some odd numbers as cosmological parameters that I will take care when I get connected to someone from the simulations group.